deltachat-ffi/Cargo.toml

```diff
@@ -26,7 +27,8 @@ thiserror = "1"
 rand = "0.7"
 
 [features]
-default = ["vendored"]
+default = ["vendored", "jsonrpc"]
```

Comment:
disabling by default and enable on desktop is probably more straight forward - otherwise we have to figure out how to disable it on android/ios, so double work :)